src/test_l1_dsDisplay.c

```diff
@@ -752,41 +724,44 @@ void test_l1_dsDisplay_negative_dsGetEDIDBytes(void) {
         return;
     }
 
-    // Step 03: Loop through all valid ports
-    for (int i = 0; i < sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]); i++) {
-        result = dsGetDisplay(kSupportedPortTypes[i], i, &displayHandle);
+    // Step 03: Obtain the display handle with valid port
+    uint32_t numPorts = UT_KVP_PROFILE_GET_UINT32("dsDisplay/Number_of_ports");
+    uint32_t videoPorts = UT_KVP_PROFILE_GET_UINT32("dsDisplay/Video_Ports");
+    for (size_t i = 0; i < numPorts; i++) {
+        uint32_t vType = videoPorts;
+        result = dsGetDisplay(vType, 0, &displayHandle);
         UT_ASSERT_EQUAL(result, dsERR_NONE);
         if(result != dsERR_NONE)
```

Comment:
remove if condition or the return statement, if debug message required

Reply:
done